migrations/versions/9ea00075af3f_.py

```diff
@@ -0,0 +1,61 @@
+"""empty message
```

Comment:
It's nice to put a message in here as it shows up in the console output when performing a migration.